src/ni/vsbuild/Pipeline.groovy

```diff
@@ -10,6 +10,7 @@ class Pipeline implements Serializable {
 
    def script
    PipelineInformation pipelineInformation
+   private String jsonConfig
```

Comment:
Do we want to prefix this with an underscore or somehow indicate that it should not be read directly, and that `geetJsonConfig()` should be called instead?

Reply:
A comment saying this is only an instance variable so we're not parsing multiple times and not to use it directly would be sufficient/